1.19.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+g++ pleakyReLU.cpp -o pleakyReLU -I/usr/include/python3.9 -I./env/lib/python3.9/site-packages/numpy/core/include -lpython3.9                          
+*/
+
+#include <iostream>
+#include <vector>
+#include <functional>
+#include <boost/math/differentiation/finite_difference.hpp>
```

Comment:
don't frogthen me. You use boost for calculating derivative from linear function???!!!